protocol/xmrmaker/watcher.go

```diff
@@ -40,7 +41,18 @@ func (s *swapState) runContractEventWatcher() {
 				log.Debugf("EventETHRefunded sent, returning from event watcher")
 				return
 			}
+		case l := <-s.logClaimedCh:
+			eventSent, err := s.handleClaimedLogs(&l)
+			if err != nil {
+				log.Errorf("failed to handle claim logs: %s", err)
+			}
```

Comment:
Was there supposed to be a return above?

Reply:
added a comment